src/test/scala/org/broadinstitute/dsde/workbench/sam/api/ResourceRoutesSpec.scala

```diff
@@ -329,7 +329,7 @@ class ResourceRoutesSpec extends FlatSpec with Matchers with ScalatestRouteTest
       status shouldEqual StatusCodes.NoContent
     }
 
-    val testUser = CreateWorkbenchUser(WorkbenchUserId("testuser"), defaultGoogleSubjectId, WorkbenchEmail("testuser@foo.com"))
+    val testUser = CreateWorkbenchUser(WorkbenchUserId("testuser"), defaultGoogleSubjectId, WorkbenchEmail("testuser@foo.com"), None)
```

Comment:
Could we define a testUser variable once for this entire file? I don't think the tests care what the id or email strings are.
Ditto other files with lots of mechanical CreateWorkbenchUser changes.
Not going to block the review over, but I think it's worth doing in general and while we're touching a hundred lines anyway.